include/boost/geometry/algorithms/detail/is_simple/linear.hpp

```diff
@@ -261,16 +261,16 @@ struct is_simple_multilinestring
 {
 private:
     template <typename Strategy>
-    struct per_linestring
+    struct not_simple
     {
-        per_linestring(Strategy const& strategy)
+        not_simple(Strategy const& strategy)
```

Comment:
But this one should be: `simple`
We shouldn't add many structures with `not_XXX`. They should be called `XXX` then they are reusable.